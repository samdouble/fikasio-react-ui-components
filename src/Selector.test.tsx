import React from 'react';
import { render } from '@testing-library/react';
import { Selector } from '.';

describe('Selector', () => {
  it('Renders correctly', () => {
    const { baseElement } = render(<Selector />);
    expect(baseElement).toMatchSnapshot();
  });
});
