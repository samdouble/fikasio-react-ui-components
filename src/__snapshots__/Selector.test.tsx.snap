// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selector Renders correctly 1`] = `
<body>
  <div>
    <span
      class="fikasio-selector fikasio-theme-dark"
    >
      <span
        style="bottom: 2px; padding-left: 5px; position: relative;"
      >
        +
      </span>
    </span>
  </div>
</body>
`;
